src/main/kotlin/app/revanced/patches/myexpenses/misc/pro/fingerprints/IsEnabledFingerprint.kt

```diff
@@ -0,0 +1,9 @@
+package app.revanced.patches.myexpenses.misc.pro.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+
+object IsEnabledFingerprint : MethodFingerprint(
+    "Z",
+    strings = listOf("feature", "feature.licenceStatus"),
+    customFingerprint = { methodDef -> methodDef.definingClass.endsWith("Liv/l;") }
```

Comment:
The class name could change over versions of the app. You should probably use something else in this fingerprint.

Reply:
I removed it. The fingerprint works without it, there is currently no method with both strings.